docs/source/reference/index.rst

```diff
@@ -64,6 +64,15 @@ MXNet
 
    optuna.integration.MXNetPruningCallback
 
+scikit-optimize
+---------------
+
+.. autosummary::
+   :toctree: generated/
+   :nosignatures:
+
+   optuna.integration.SkoptSampler
```

Comment:
The documentation should be generated from `optuna_integration`, not `optuna.integration` .
```suggestion
   optuna_integration.SkoptSampler
```

Reply:
I wonder why `SkoptSampler` is different from other modules, e.g., `optuna.integration.MXNetPruningCallback` in the 65th line.

Reply:
I think [the documentation in the current main branch](https://github.com/optuna/optuna-integration/blob/main/docs/source/reference/index.rst?plain=1) is generated from `optuna_integration`.
Is your branch up-to-date?

Reply:
You're right. I updated this branch and fixed it.